service/src/main/java/org/apache/celeborn/server/common/service/config/ConfigLevel.java

```diff
@@ -20,4 +20,5 @@
 public enum ConfigLevel {
   SYSTEM,
   TENANT,
+  TENANT_USER
```

Comment:
IMO, we could extend the `TENANT` config instead of `TENANT_USER` introduction. The difference between `TENANT` and `TENANT_USER` is that `TENANT_USER` has `name` identifier, therefore the config level of `TENANT_USER` is also `TENANT`.

Reply:
> IMO, we could extend the `TENANT` config instead of `TENANT_USER` introduction. The difference between `TENANT` and `TENANT_USER` is that `TENANT_USER` has `name` identifier, therefore the config level of `TENANT_USER` is also `TENANT`.

 if we identify the config belongs to tenant or user just use the `name` identifier is empty or not  may be a little trick, so I prefer add a new ConfigLevel for User